transit_predictions_511.py

```diff
@@ -309,7 +96,7 @@ def get_predictions(self):
 
                     route_predictions = ' '.join(route.predictions[:n])
 
-                    prediction_text.append(route_predictions)
+                    prediction_text.append(f'{line_text} {route_predictions}')
```

Comment:
This changes the example now from:

(Line Number) (Line Name)
(Predictions)

to

(Line Number) (Predictions)

Anyone can go into this file to change the code for the controller or write their own to change this format for lines.

A next step might be to write a formatter so that this can be called instead and customization is simpler.